src/components/BookPage/AddReview/index.js

```diff
@@ -0,0 +1,119 @@
+import { useEffect, useState } from "react";
+import "./addreview.css";
+
+function Form(props) {
```

Comment:
The name form is way to generic. Call it something like AddReviewForm

Reply:
done